leftwm/src/bin/leftwm-check.rs

```diff
@@ -279,6 +303,10 @@ fn check_theme_toml(filepath: PathBuf, verbose: bool) -> Result<PathBuf> {
                 if verbose {
                     println!("The theme file looks OK.");
                 }
+                println!(
+                    "\x1b[1;93mWARN: TOML as config format is about to be deprecated.
```

Comment:
this `\x1b[1;93` is for being printed out with a color right? Why not choosing a crate for doing this?